wgsl/index.bs

```diff
@@ -9807,19 +9821,21 @@ The rules for analyzing expressions take as argument both the expression itself
       <td>
       <td class="nowrap">*CF*, *CF*
       <td>
-   <tr><td>identifier [=resolves|resolving=] to function-scope variable "x"
-      where the [=load rule=] is invoked during [=type checking=] with "x" as
-      [=root identifier=] of the [=memory view=]
+   <tr><td>identifier [=resolves|resolving=] to function-scope variable "x",
+      where the identifier appears as the [=root identifier=] of a [=memory view=]
+      expression, *MVE*, and the [=load rule=] is invoked on *MVE* during
+      [=type checking=]
       <td>*Result*
       <td class>*X* is the node corresponding to the value of "x" at the input to the statement containing this expression
       <td class="nowrap">*CF*, *Result*
       <td class>*Result* -> {*CF*, *X*}
 
       Note: *X* is equivalent to *Vout*(*prev*) for "x"<br>
       (see [[#func-var-value-analysis]])
-   <tr><td>identifier [=resolves|resolving=] to function-scope variable "x"
-      where the [=load rule=] is not invoked during [=type checking=] with "x" as
-      the [=root identifier=] of the [=memory view=]
+   <tr><td>identifier [=resolves|resolving=] to function-scope variable "x",
+      where the identifier appears as the [=root identifier=] of a [=memory view=]
+      expression, *MVE*, and the [=load rule=] is not invoked on *MVE* during
```

Comment:
nit:  Might be worth emphasizing the "not", either italics or bold.

Reply:
An example suggested from internal review:

Consider   `let p = &y[x[10]];`

In this case:

* one MVE is `x[10]` and it *does* have the load rule applied.  The root identifier is `x`.
* the other MVEs are `y[x[10]]` and `&y[x[10]]`, with root identifier `y`, but neither of those have the load rule applied during type checking.